modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -227,18 +227,22 @@ export default class Tile3DLayer extends CompositeLayer {
 
   _makeSimpleMeshLayer(tileHeader, oldLayer) {
     const content = tileHeader.content;
-    const {attributes, modelMatrix, cartographicOrigin, texture} = content;
-    const {positions, normals, texCoords} = attributes;
+    const {modelMatrix, cartographicOrigin, texture} = content;
+    const attributes = {};
+    attributes.positions = content.attributes.positions;
+    attributes.positions.value = new Float32Array(attributes.positions.value);
+    if (content.attributes.normals) {
+      attributes.normals = content.attributes.normals;
+    }
+    if (content.attributes.texCoords) {
+      attributes.texCoords = content.attributes.texCoords;
+    }
 
     const geometry =
       (oldLayer && oldLayer.props.mesh) ||
       new Geometry({
         drawMode: GL.TRIANGLES,
-        attributes: {
-          positions: {...positions, value: new Float32Array(positions.value)},
```

Comment:
With `new Float32Array` I just kept existing logic.

Reply:
Fair enough. Looks like a hack made to get things to work in a previous PR. Let's fix it separately.